architecture/README.md

```diff
@@ -0,0 +1,159 @@
+# TiDB Community Architecture
+
+Open source projects live and breathe the open source community with different
+types of roles. The TiDB community has the following organizations:
+
+![TiDB Community Architecture](../media/architecture.svg)
+
+## Content
+
+<!-- vim-markdown-toc GFM -->
+
+* [PMC](#pmc)
+* [TiDB Developer Group](#tidb-developer-group)
+    * [Contributor](#contributor)
+        * [How to become a Contributor?](#how-to-become-a-contributor)
+        * [Contributor List](#contributor-list)
+    * [Active Contributor](#active-contributor)
+    * [Reviewer](#reviewer)
+        * [How to become a Reviewer?](#how-to-become-a-reviewer)
+        * [Responsibilities and Privileges](#responsibilities-and-privileges)
+    * [Committer](#committer)
+        * [How to become a Committer?](#how-to-become-a-committer)
+        * [Responsibilities and Privileges](#responsibilities-and-privileges-1)
+        * [Definition of Inactive Committer and operations:](#definition-of-inactive-committer-and-operations)
+        * [Committer List](#committer-list)
+    * [Maintainer](#maintainer)
+        * [How to become a Maintainer?](#how-to-become-a-maintainer)
+        * [Responsibilities and Privileges](#responsibilities-and-privileges-2)
+* [TiDB User Group](#tidb-user-group)
+* [Organizer Committee](#organizer-committee)
+
+<!-- vim-markdown-toc -->
+
+## PMC
+
+TBD
+
+## TiDB Developer Group
+
+### Contributor
+
+A Contributor refers to the person who contributes to the following projects:
+
+* TiDB: https://github.com/pingcap/tidb
+* Parser: https://github.com/pingcap/parser
+* TiKV: https://github.com/tikv/tikv
+* TiSpark: https://github.com/pingcap/tispark
+* PD: https://github.com/pingcap/pd
+* Docs: https://github.com/pingcap/docs
+* Docs-cn: https://github.com/pingcap/docs-cn
+
+#### How to become a Contributor?
+
+If a PR (Pull Request) submitted to any one of the above projects by you is
+approved and merged, then you become a TiDB Contributor.
+
+You are also encouraged to participate in the projects in the following ways:
+
+* Actively answer technical questions asked by community users.
```

Comment:
Shall we explain where they can answer questions from users?

Reply:
how about adding a URL link to Ask TUG?

Reply:
Sounds great!

Reply:
Oh, no. It should be on the github issues, not Ask TUG.

Reply:
@Soline324 updated, PTAL again.